### DISCUSSION TITLE
Multiplexer connection.

### DISCUSSION THREAD

**Discussion by Opekusha** [marcel-licence/esp32_basic_synth#25] [Q&A]
Please tell me which Pin of ESP32 the output from the multiplexer is connected to.

**Comment by marcel-licence** [comment]
```
#define ADC_MUL_S0_PIN	23 // pin 23 for S0 
#if ADC_INPUTS > 2
#define ADC_MUL_S1_PIN	18 // pin 18 for S1 
#endif
#if ADC_INPUTS > 4
#define ADC_MUL_S2_PIN	14 // pin 14 for S2
#endif
#if ADC_INPUTS > 8
#define ADC_MUL_S3_PIN	5	 /* this has not been tested */
#endif
```

`    adcAttachPin(12); // pin 12 is used for ADC input`

S3 goes to ground so you can use C0..C7 of the multiplexer.
EN should be also connected to ground.
SIG to pin 12 of the Audio Kit V2.2

Hope this helps

**Comment by Opekusha** [comment]
I am using the first option with a PCM external DAC. I do not have an audio KIT. You need a pin for the ESP32 G *. The rest of the pins are understandable, but thanks anyway for your attention.

**Comment by marcel-licence** [comment]
I've bring back the possibility to compile for the ESP32 DOIT DEVKIT. I think that will help. I've updated the readme. Please tell me if something missing

**Comment by tiretrak** [comment]
![1](https://user-images.githubusercontent.com/7010744/118988994-ba7c3c80-b992-11eb-9de4-d109988b41cb.jpg)

**Comment by marcel-licence** [comment]
Thanks for the picture, it's looking great!
